<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSimplifierClass</class>
 <widget class="QMainWindow" name="ProjectSimplifierClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProjectSimplifier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <layout class="QVBoxLayout" name="ProgramTitleLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="sLAB_ProgramTitle">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;VS项目瘦身&lt;/span&gt;：通过删除项目编译或运行调试时产生的中间项和pdb文件，来缩小项目体积&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="StatusShowLayout">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="LAB_ShowTips">
        <property name="text">
         <string>Content</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="ExecutionLayout">
      <property name="leftMargin">
       <number>120</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="BTN_executeProcess">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>执行清理</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="PathSelectionLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="sLAB_Dir">
        <property name="text">
         <string>项目目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BOX_InputFilePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BTN_OpenDir">
        <property name="text">
         <string>打开目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QGridLayout" name="ShowTreeLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="LAB_PrimitiveSize">
        <property name="text">
         <string>未选定目录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTreeWidget" name="BOX_PrimitiveDirectView">
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>大小</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="4" colspan="3">
       <widget class="QTreeWidget" name="BOX_ProcessedDirectView">
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>大小</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sLAB_Primitive">
        <property name="text">
         <string>原始大小：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="sLAB_Processed">
        <property name="text">
         <string>清理后大小：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="LAB_ProcessedSize">
        <property name="text">
         <string>未选定目录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="LAB_ShowRemovableSize">
        <property name="text">
         <string>Content</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProjectSimplifier.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BTN_executeProcess</sender>
   <signal>clicked()</signal>
   <receiver>ProjectSimplifierClass</receiver>
   <slot>executeSimpifyDirect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_OpenDir</sender>
   <signal>clicked()</signal>
   <receiver>ProjectSimplifierClass</receiver>
   <slot>GetDirect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>executeSimpifyDirect()</slot>
  <slot>GetDirect()</slot>
 </slots>
</ui>
